src/design-system/components/icon-button/IconButton.tsx

```diff
@@ -0,0 +1,63 @@
+import { CircularProgress, IconButtonProps } from '@mui/material'
+import React, { useState } from 'react'
+import { IconButtonVariant } from '~/types'
+import AddRoundedIcon from '@mui/icons-material/AddRounded'
+import { IconButton as MuiIconButton } from '@mui/material'
+import './IconButton.scss'
+interface S2SIconButtonProps extends Omit<IconButtonProps, 'size'> {
```

Comment:
Add empty line between imports and interface declaration

```suggestion

interface S2SIconButtonProps extends Omit<IconButtonProps, 'size'> {
```

Reply:
Thanks, didn`t notice it!